parallel/utils.py

```diff
@@ -0,0 +1,9 @@
+import os
+from glob import glob
+
+
+def get_gpu_count() -> int:
+    if not os.path.isdir("/dev/"):
+        raise RuntimeError("No /dev/ directory, can't detect number of GPUs")
+    return len(glob("/dev/nvidia[0-9]*"))
```

Comment:
This is what I have had the most success in using to determine GPU count, though if someone knows a preferable way let me know

Reply:
```
nvidia-smi -L
```
is probably more preferred as 
1) it checks to make sure that drivers are present and functional (grepping dev just checks PCIE slots IIRC)
2) generates UUIDs for the GPUs in case they're useful

Reply:
@proteneer Running `nvidia-smi -L` seems to take about 8 seconds to run on a p3.16xlarge, is that acceptable? It does verify that drivers exist, however it will still die if cuda is incorrect configured.

Reply:
That's fine - it's negligible relative to the MD simulation.